python/cudf/cudf/dataframe/datetime.py

```diff
@@ -49,6 +49,14 @@ def __init__(self, **kwargs):
         assert self.dtype.type is np.datetime64
         self._time_unit, _ = np.datetime_data(self.dtype)
 
+    def __contains__(self, item):
+        try:
+            item = pd.to_datetime(item)
+            item = columnops.as_column(item).as_numerical[0]
```

Comment:
This is causing an 8 byte host --> device copy, followed by an 8 byte device --> host copy unnecessarily. We should be able to handle creating the int64 of the datetime scalar that we're looking for here on the host. I believe `item = pd.to_datetime(item).to_numpy().view('int64')` should work for getting an int64 scalar (which I believe is the intention here.